src/SendGrid/Helpers/Errors/Model/SendGridErrorResponse.cs

```diff
@@ -6,9 +6,10 @@
     public class SendGridErrorResponse
     {
         /// <summary>
-        /// Gets or sets the error Status Code
+        /// Gets or sets the error Status Code.<br/>
+        /// Might be any <see cref="System.Net.HttpStatusCode"/>, not just ones in <see cref="SendGridStatusCode"/>.
```

Comment:
What happens to this property when the value is not defined in `SendGridStatusCode`? Guessing `null` and worried that clients would then not get the actual status code. Could this be added as an extra property of `SendGridErrorResponse` so it's backwards-compatible and supports the benefits of the enum?

Reply:
`enum`s are value type, so cannot be `null`. If value of `ErrorHttpStatusCode` is not defined in `SendGridStatusCode`, it will still be stored in `ErrorHttpStatusCode` (Visual Studio will just show you int value instead of a name).

Admittedly, this might be not the best solution since value of `ErrorHttpStatusCode` might be outside of what's defined `SendGridStatusCode`, but no information will be lost.